setup.py

```diff
@@ -27,12 +29,13 @@
         'License :: OSI Approved :: MIT License',
     ],
     keywords='pydocstyle, PEP 257, pep257, PEP 8, pep8, docstrings',
+    packages=('pydocstyle',),
+    py_modules=('main',),
```

Comment:
Have you investigated how this is actually installed on a person's computer though? I think this might not do exactly what you want.